install.sh

```diff
@@ -319,7 +295,45 @@ unzip() {
 }
 
 get_default_install_dir() {
-    printf %s "${HOME}/.temporalio"
+    printf %s "${HOME}/.temporalio/bin"
+}
+
+get_install_dir() {
+    local _dir
+    _dir="$(get_default_install_dir)"
+
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            --dir)
+                _dir="$2"
+                shift
+                ;;
+            *)
+                ;;
+        esac
+        shift
+    done
+
+    printf %s "$_dir"
+}
+
+prompt_for_path() {
```

Comment:
I'd replace this entire thing with:

```
say "Temporal CLI installed at $_dir/temporal"
if echo ":$PATH:" | grep -q "$_dir"; then
    say "Start the server with: temporal server start-dev"
    say "Or start a workflow with: temporal workflow start"
    say "For usage, run: temporal --help"
else
    say "For convenience, we recommend adding it to your PATH"
    say "If using bash, run echo "export PATH=\"$PATH:$_dir\" >> ~/.bash_profile"
fi
```

And maybe add the "getting started" commands to the "else" statement too.

Reply:
done